### ISSUE TITLE
Keyboard navigation during debugging

### ISSUE THREAD

**Issue by TomaszCz** [flutter/devtools#7264]
I cannot find a way to use keyboard navigation during debugging. I guess that could be very nice to have.
Simple F8 ( run ) F10 ( step in ) , F11 ( step over )
![image](https://github.com/flutter/devtools/assets/20865719/220e7ca2-a0a3-4a7f-90cb-02965da3bd09)
___
**DevTools version**: 2.31.1
**Connected Device**:
CPU / OS: Web windows
Connected app type: Flutter web (debug build)
Dart Version: 3.3.0
Flutter Version: 3.19.0 / stable
Framework / Engine: bae5e49bc2 / 04817c99c9

**Comment by kenzieschmoll** [comment]
closing as a duplicate of https://github.com/flutter/devtools/issues/3867
